RFC-004-Bitcoin.md

```diff
@@ -0,0 +1,54 @@
+# Bitcoin Definitions
+
+As per [Bitcoin Core](https://github.com/bitcoin/bitcoin/) concensus.
+
+Introduced for RFC-003 (should RFC-003 be part of the title?)
+
+## Ledger definition
+
+Taken from registry
```

Comment:
I would like it if we do it the other way round.

I know that the registry already has some values in there but it would be good if this RFC would officially _add_ them.

Reply:
This means that we import the detailed values from the registry to here (as done with the table you can see in teh file).
Yes, in the registry we would remove some of the details and only keep generic definition and reference back to this new RFC.